### PR TITLE
htop: add page

### PR DESCRIPTION
**Pull request by khezam**
- [ +] The page (if new), does not already exist in the repo.
- [ +] The page is in the correct platform directory (`common/`, `linux/`, etc.)
- [ +] The page has 8 or fewer examples.
- [ +] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/master/CONTRIBUTING.md#commit-message).
- [ +] The page follows the [content guidelines](/tldr-pages/tldr/blob/master/CONTRIBUTING.md#guidelines).
- [ +] The page description includes a link to documentation or a homepage (if applicable).

### CODE REVIEW

**Comment by andrik** [comment]
Hi @khezam,

Thank you for your submission.

I would suggest you merge these page with [these one](https://github.com/tldr-pages/tldr/blob/master/pages/linux/htop.md)

I noticed that you put the page under common directory and make sense as the developers of htop said: 
"It is developed primarily on Linux, but we also have code for running under FreeBSD and Mac OS X (help and testing are wanted for these platforms!)"

 So, recapping:

1) Merge the new examples with the existing page of htop and update the PR
2) Submit a new PR removing the htop from linux folder

File: pages/common/htop.md [hunk 1]
```diff
@@ -0,0 +1,16 @@
+# htop
+
+> An interactive process viewer for Unix.
+> More Information: <https://hisham.hm/htop/>
```
Comment (Aracki) [inline pages/common/htop.md]: I would maybe use this link: https://linux.die.net/man/1/htop (more general one)

**Comment by Aracki** [comment]
@khezam  The page already exists for Linux platform. You can run it with `tldr htop -p linux`. 

If the command can be used within all platforms I would suggest we have one page and put that in the `/common` dir.

**Comment by schneiderl** [comment]
I would say we do what @andrik says, given what the command developer says.

**Comment by agnivade** [comment]
> I would say we do what @andrik says, given what the command developer says.

Yep let's do that.

File: pages/common/htop.md [hunk 2]
```diff
@@ -0,0 +1,16 @@
...
+
+- View the list of processes:
+
+`htop`
+
+- View the list of processes in the tree view: 
+
+`htop -t`
```
Comment (einverne) [inline pages/common/htop.md]: I think it should be mentioned `-t` flag is new in htop 2.2.0.

**Comment by sbrl** [comment]
Hey! It looks like we've already got a page for `htop` in the `linux` platform. Perhaps this page could be merged with that one?

**Pull request closed without merging**